core/src/main/resources/error/error-classes.json

```diff
@@ -70,6 +70,11 @@
     ],
     "sqlState" : "23000"
   },
+  "ELEMENT_AT_BY_INDEX_ZERO" : {
+    "message" : [
+      "The index 0 is invalid. The indexes can be < 0 and > 0 (the first element has index 1)."
```

Comment:
`and` might confuse users. Could you write this in more straight way:
"... An index shall be either < 0 or > 0 (the first element has index 1)."

Reply:
Updated